website/docs/community/people.md

```diff
@@ -4,4 +4,16 @@ title: People
 
 # Project Team
 
-This is content
+## Liza Babu
+
+![Liza Babu](../../static/img/people/liza_babu.jpeg)
+
+Liza is a contributor to the OpenEdu project (21-COP-0016).
+She contributes beyond the scenes with ideas to the methodology of the project and takes care of the project overall presentation.
+
+Liza finished her Masters degree in Advanced Cybersecurity in 2019 at University POLITEHNICA of Bucharest, Romania.
+She is also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in Computer Programming, Operating Systems and Hardware-Software Interface classes.
```

Comment:
I'd say without `the`.

Reply:
I agree, I left it here to see your opinion (I saw Teo's comment on your PR).